### PR TITLE
Configure base 

### PR DESCRIPTION
**Pull request by filwas**

### CODE REVIEW

File: src/components/icons/CarouselArrow.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+interface CarouselArrowProps {
+  className?: string;
+  direction: "left" | "right";
```
Comment (Sumick) [inline src/components/icons/CarouselArrow.tsx:5]: **nitpick:** It also could be an enum, so you don't repeat string values

File: src/components/modularForm/Ingredient.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        <div className={nameStyle}>{props.name}</div>
+        {selectorHelper(props.selector, props.options)}
+      </div>
+    );
+  }
+}
+
+export default Ingredient;
+
+function selectorHelper(selector: string, options: string[]) {
```
Comment (Sumick) [inline src/components/modularForm/Ingredient.tsx:82]: **nitpick:** Selector should be an enum type - you already created it

File: src/components/selectorComponents/Carousel.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+          : prevOptionIndex - 1
+      );
+    } else if (direction == "right") {
+      setOptionIndex((prevOptionIndex) =>
+        prevOptionIndex === props.carouselOptions.length - 1
+          ? 0
+          : prevOptionIndex + 1
+      );
+    } else {
+    }
```
Comment (Sumick) [inline src/components/selectorComponents/Carousel.tsx:32]: **nitpick:** Redundant else

File: src/components/selectorComponents/Checkbox.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    const newState = !state;
+    setState(newState);
+
+    if (props.onStateChange) {
+      props.onStateChange(newState);
+    }
+  };
+  return (
+    <div className={styles.checkboxWrapper} onClick={clickHandler}>
+      {state && <div className={styles.innerSquare} />}
```
Comment (Sumick) [inline src/components/selectorComponents/Checkbox.tsx:22]: **nitpick:** I strongly suggest to put `input` of type `checkbox` with `hidden` attribute, so you keep with semantics. Currently it is not clear for example for screen readers that this component is Checkbox

**Review by Sumick — APPROVED** [review]
Everything is split nicely. It's clean, readable, maintainable. I love it <3 Amazing job. I've left few non blocking comments, so you can consider before merging.

File: src/components/selectorComponents/Checkbox.tsx (hunk 4 above)
Comment (filwas) [reply src/components/selectorComponents/Checkbox.tsx:22]: I'll apply this in the next PR, because I made changes there to the Checkbox element.

**Pull request merged**